search-lambda/CKS.SearchLambda/http-api-v2-request.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Is this a sample file for testing this? Can we have something in the readme that explains what this is/how to use it?

Reply:
change the url object to have a search query string....

Reply:
add info to readme about what this file does